### ISSUE TITLE
Pre-trained weights for Unet

### ISSUE THREAD

**Issue by gcunhase** [NVIDIA/DeepLearningExamples#963]
Does anyone know where I can download the pre-trained weights for [3D and 2D Unet](https://github.com/gcunhase/DeepLearningExamples/tree/master/TensorFlow2/Segmentation)?

Thank you in advance,
Gwena

**Comment by mmarcinkiewicz** [comment]
Hello @gcunhase!
You can find a checkpoint for 
- U-Net Medical TF2 (2D) [here](https://ngc.nvidia.com/catalog/models/nvidia:unet_medical_tf2_amp_ckpt/files) 
- U-Net3D [here](https://ngc.nvidia.com/catalog/models/nvidia:unet3d_tf_amp_ckpt)

let me know if that helps.

**Comment by gcunhase** [comment]
Thank you, that helps! Do you know where I can find similar checkpoints for Mask R-CNN? I can only find weights for the ResNet backbone.

**Comment by mmarcinkiewicz** [comment]
@gcunhase, we are trying to differentiate between performance and accuracy - performance is the speed of training/inference and accuracy is the quality metric used to determine the convergence. I assume that you are interested in accuracy - in case of Unet we use `eval_dice_score` as a quality metric. The weights that you loaded give `eval_dice_score = 0.8934073820710182`, which is indeed the target accuracy that we are reporting in the readme. Does that answer your question?

Regarding MaskRCNN - please open another issue.
